docs/product_release.md

```diff
@@ -0,0 +1,59 @@
+Product releases are done by creating a GitHub release. Every release is following the [semantic versioning](https://semver.org/) pattern and must be marked in the git history via tag.
```

Comment:
in general: pay attention to CAPITAL or small letters => especially when referencing URLs and documents

Reply:
You mean Semantic Versioning? or something else?

Reply:
thanks for checking / resolved